document/endnotes_test.go

```diff
@@ -0,0 +1,285 @@
+package document_test
```

Comment:
https://github.com/unidoc/unioffice/wiki/UniOffice-Developer-Guide#7-license-header-in-each-go-file